src/v/cluster/types.h

```diff
@@ -174,6 +174,21 @@ inline std::error_code make_error_code(tx_errc e) noexcept {
     return std::error_code(static_cast<int>(e), tx_error_category());
 }
 
+using kafka_offset = named_type<int64_t, struct kafka_offset_type>;
```

Comment:
can this be `kafka::offset`?

Reply:
oh I see this is a symptom of us needing some refactoring. makes sense

Reply:
made it kafka::offset